src/responder/ifp/ifp_users.c

```diff
@@ -584,6 +584,12 @@ ifp_users_find_by_name_and_cert_send(TALLOC_CTX *mem_ctx,
     }
 
     if (!SBUS_REQ_STRING_IS_EMPTY(pem_cert)) {
+        state->pem_cert = talloc_strdup(state, name);
```

Comment:
I'm sorry to delay an important patch, but shouldn't this strdup pam_cert instead of name? Looking at the request, it probably doesn't matter in the end, because the request doesn't check the state->pam_cert contents, but only checks it for being null/non-null, but it still looks like a bug to me.

Reply:
bummer, you are completely right, copy-and-paste: 0 - code-review: 1 :-)

New version pushed.